### ISSUE TITLE
Release Twisted 16.3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8471]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8471|
|Type|task|
|Created|2016-06-14 12:25:50Z|
|Branch|https://github.com/twisted/twisted/tree/release-16.3.0-8471|

Beep boop, lets push out those releases. Some good post-PyCon changes in here.

<details><summary>Searchable metadata</summary>

```
trac-id__8471 8471
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__Twisted_16_3 Twisted 16.3
branch__release_16_3_0_8471 release-16.3.0-8471
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__None None
time__1465907150661763 1465907150661763
changetime__1467808742887016 1467808742887016
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

One of the builders crashes, made [#8563](https://github.com/twisted/twisted/issues/8563)#ticket to fix it. But it should be fine for now.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

Builders spun, please review.

This causes some failures on the static/twistedchecker checkers; these can be safely ignored. The static builders are failing because of [#8563](https://github.com/twisted/twisted/issues/8563) and because the branch that removed the rest of the pyflakes errors is post-this branch.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Looks good. Thanks!

Please release!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset c930bbbf25085648058e7837c030eab8c1e07b39
```
#!CommitTicketReference repository="" revision="c930bbbf25085648058e7837c030eab8c1e07b39"
Merge release-16.3.0-8471: Release Twisted 16.3

Author: hawkowl
Reviewer: adiroiban
Fixes: #8471
```
